go/exorelay/exorelay_test.go

```diff
@@ -16,26 +16,26 @@ import (
 	"github.com/Originate/exocom/go/structs"
 )
 
+func initializeExocom() *exocomMock.ExoComMock {
+	exocom := exocomMock.New()
+	go func() {
+		err := exocom.Listen(4100)
+		if err != nil {
+			log.Fatal(err)
+		}
+	}()
+	defer exocom.Close()
+	return exocom
+}
```

Comment:
Couldn't end up using a new exocom per scenario as you get the error "Server closed". Went with this and reset the received messages every test.